src/Sia.Gateway/Filters/EventFilters.cs

```diff
@@ -6,10 +6,18 @@
 using System.Text;
 using Microsoft.Extensions.Primitives;
 using Newtonsoft.Json;
+using Sia.Domain;
+using AutoMapper;
 
-namespace Sia.Data.Incidents.Filters
+namespace Sia.Gateway.Filters
 {
+<<<<<<< HEAD:src/Sia.Gateway/Filters/EventFilters.cs
+    public class EventFilters 
+        : IFilters<Data.Incidents.Models.Event>,
+        IFilters<Domain.Event>
+=======
     public class EventFilters : Filters<Event>
+>>>>>>> master:src/Sia.Data.Incident/Filters/EventFilters.cs
```

Comment:
Merge conflict... looks like it happened during the merge from master (71ec2b22b7808321ce115f2d1fd99875f6ac24e2)